src/main/java/com/auction/auction/bid/controller/AuctionController.java

```diff
@@ -13,19 +13,15 @@
 @RestController
 @RequestMapping("/auctions")
 public class AuctionController {
+
     @Autowired
-    private AuctionService auctionService;
```

Comment:
Usually "Interface" is not used because it must be and interface by default. So best naming is "AuctionService" and everyone knows it is an interface. You should not change it, it is just info for future